src/DynamoMLDataPipeline/DynamoMLDataPipeline.csproj

```diff
@@ -0,0 +1,64 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+		<UILib>true</UILib>
+  </PropertyGroup>
+  <ImportGroup Label="PropertySheets">
+		<Import Project="$(SolutionDir)Config\CS_SDK.props" />
+  </ImportGroup>
+  <PropertyGroup>
+	<Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+	<Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+	<ProjectGuid>{5DF79F45-5F2C-41C1-BACC-890AE514CDA8}</ProjectGuid>
+	<OutputType>Library</OutputType>
+	<RootNamespace>DynamoMLDataPipeline</RootNamespace>
+	<AssemblyName>DynamoMLDataPipeline</AssemblyName>
+	<FileAlignment>512</FileAlignment>
+	<AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
```

Comment:
should not be here.

Reply:
Removed. https://github.com/DynamoDS/Dynamo/pull/15116